src/renderer/coremods/settings/pages/Addons.tsx

```diff
@@ -33,6 +33,17 @@ function getManager(type: AddonType) {
   throw new Error("Invalid addon type");
 }
 
+function getSettingsElement(id: string, type: AddonType) {
+  if (type === AddonType.Plugin) {
+    return window.replugged.plugins.plugins.get(id)?.exports?.Settings;
```

Comment:
Shouldn't settings be gotten in a different way? Could it work through the manifest? I'm not sure how I feel about it being gotten from exports...
And otherwise, if you continue using this method, `replugged.plugins.getExports` exists (and shouldn't you import this somehow)

Reply:
@colin273 said I should do it this way, I originally had a manifest key. I'll update to getExports later.